pysass.cpp

```diff
@@ -512,19 +512,24 @@ PySass_compile_string(PyObject *self, PyObject *args) {
     struct Sass_Context *ctx;
     struct Sass_Data_Context *context;
     struct Sass_Options *options;
-    char *string, *include_paths;
+    char *string, *include_paths, *source_map_file;
     const char *error_message, *output_string;
     Sass_Output_Style output_style;
-    int source_comments, error_status, precision, indented;
+    int source_comments, error_status, precision, indented,
+        source_map_embed, source_map_contents, source_map_file_urls,
+        omit_source_map_url;
     PyObject *custom_functions;
     PyObject *custom_importers;
+    PyObject *source_map_root;
     PyObject *result;
 
     if (!PyArg_ParseTuple(args,
-                          PySass_IF_PY3("yiiyiOiO", "siisiOiO"),
+                          PySass_IF_PY3("yiiyiiOO|iiiO", "siisiiOO|iiiO"),
                           &string, &output_style, &source_comments,
-                          &include_paths, &precision,
-                          &custom_functions, &indented, &custom_importers)) {
+                          &include_paths, &precision, &indented,
+                          &custom_functions, &custom_importers,
+                          &source_map_contents, &source_map_embed,
+                          &omit_source_map_url, &source_map_root)) {
```

Comment:
just noticing the order changed here -- I assume this was intentional and you've fixed the callsites but wanted to double check